src/pages/conferences/index.astro

```diff
@@ -1,5 +1,9 @@
 ---
 import Layout from '../../layouts/layout/Layout.astro';
+import {getAllConferences} from "../../service/api/conferenceAPI"
+
+// const response = await getAllConferences();
```

Comment:
remove commented code if not neeed

Reply:
we will be needing it in displaying data